### ISSUE TITLE
Improve Contrast

### ISSUE THREAD

**Issue by IgorFerreiraMoraes** [IgorFerreiraMoraes/Mignon-icon-theme#3]
### Description
Many comments pointed that the pastel colors looked "washed out" or "pale". While this is somewhat expected, since those tones are created by incorporating white in the more saturated colors, the lack of contrast makes the readability difficult.

### Steps
- [x] Check the current palette for accessibility and contrast 

- [x] Alter the colors to more saturated/darker colors

- [x] Test the colors with a sample of icons and redo the checks

- [x] Update the whole icon set to the new colors
